NAMESPACE

```diff
@@ -1,5 +1,7 @@
 # Generated by roxygen2: do not edit by hand
 
+S3method(as.data.frame,dw_data_tabulates)
+S3method(as.data.frame,dw_data_xtabulate)
```

Comment:
Do we already use this class name somewhere? 
If not, maybe this is also a good opportunity to rename it to something more informative? 
E.g. `datawizard_crosstabs`.